koku/masu/api/db_performance/db_performance.py

```diff
@@ -140,21 +138,33 @@ def get_pg_settings(self, setting_names=None):
         cur = self._execute(sql, params or None)
         return cur.fetchall()
 
-    def get_pg_engine_version(self):
-        global SERVER_VERSION
-        if not SERVER_VERSION:
-            sql = """
--- PARSED PG ENGINE VERSION
-select (boot_val::int / 10000::int)::int as "release",
-       ((boot_val::int / 100)::int % 100::int)::int as "major",
-       (boot_val::int % 100::int)::int as "minor"
-  from pg_settings
- where name = 'server_version_num';
-"""
-            res = self._execute(sql, None).fetchone()
-            SERVER_VERSION.extend(res.values())
-
-        return SERVER_VERSION
+    def get_pg_engine_version(self) -> str:
+        sql = "SHOW server_version_num;"
+        res = self._execute(sql, None).fetchone()
+        server_version_num = res["server_version_num"]
+
+        # Split the version into two part chunks
+        # Examples: 090600 --> ['09', '06', '00']
+        #           120001 --> ['12', '00', '01']
+        #           140011 --> ['14', '00', '11']
+        parts = list(map("".join, zip(*[iter(server_version_num)] * 2)))
+        parts = [int(part) for part in parts]
+
+        # The second part of the version number was only used in PastgreSQL < 10
+        # https://www.postgresql.org/support/versioning/
+        # https://www.postgresql.org/docs/release/
+        #
+        # Examples:
+        #   9.0.0, 9.1.2
+        #   10.1, 10.2
+        #
+        major = parts[0]
+        if major < 10:
```

Comment:
Will `major` ever be less than 10? The oldest supported version of postgres is 12, so we do not need to support the major.minor.patch versions of postgres.

Reply:
Nope, it won't be.

Reply:
But I figured you wanted to add better support for 9.x and earlier based on the previous comment. I didn't initially test much for versions < 10.